test/functional/feature_maxtipage.py

```diff
@@ -22,23 +22,24 @@ def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 2
 
-    def test_maxtipage(self, maxtipage, set_parameter=True):
+    def test_maxtipage(self, maxtipage, set_parameter=True, test_deltas=True):
         node_miner = self.nodes[0]
         node_ibd = self.nodes[1]
 
         self.restart_node(1, [f'-maxtipage={maxtipage}'] if set_parameter else None)
-        self.connect_nodes(0, 1)
-
-        # tips older than maximum age -> stay in IBD
+        self.connect_nodes(0, 1, timeout=5)
```

Comment:
Can you explain why this is changed? If you locally want to change the timeout, you can use `--timeout-factor`. Changing it in the code may result in CI sanitizer timeouts.

Reply:
If the test fails, it'll hang for a while and lock up CI. But if the convention is to rely on `--timeout-factor` and kind of disregard CI/local default experience, I can revert.

Reply:
Bumped this to `10`, which I would think would be long enough to not cause spurious failures given such a simple test?

Reply:
Yeah, it is probably fine. It seems to be pulling in a bunch of mypy refactorings, which I don't like too much.